### ISSUE TITLE
[Bug Report] Kayboard opens in capslock mode when opened for the first time in certain apps

### ISSUE THREAD

**Issue by 0karin0** [futo-org/android-keyboard#202]
Steps to reproduce:
- Open instagram with FUTO Keyboard as default keyboard
- Open a direct message chat
- Keyboard will open in capslock mode without showing the capslock indicator on shift key.

Expected behaviour: 
Keyboard never defaults to capslock mode on opening.

Version: 0.1.22
Version Code: 10804

**Comment by The-Quantum-Alpha** [comment]
I have the same experience. Sometimes it is stuck in cap locks, sometimes it jitters.

**Comment by ejerke** [comment]
I have this issue too, pretty sure it doesn't always need to be first time in those apps.

**Comment by 0karin0** [comment]
Yeah, I've observed the issue much more since filing this bug report, it doesn't seem to be correlated with specific applications, it just happens randomly. 

It's more common on the first time I open the keyboard in a newly spawned text-box. E.g. moving from the list of current SMS conversations into a particular conversation, then opening the keyboard to type a response, or opening the keyboard to write a reddit post/comment.

**Comment by 0karin0** [comment]
Sorry, accidentally fat-fingered that on my phone, hit "close with comment" instead of "comment". Here's the original comment:

Yeah, I've observed the issue much more since filing this bug report, it doesn't seem to be correlated with specific applications, it just happens randomly.

It's more common on the first time I open the keyboard in a newly spawned text-box. E.g. moving from the list of current SMS conversations into a particular conversation, then opening the keyboard to type a response, or opening the keyboard to write a reddit post/comment.

**Comment by abb128** [comment]
If you're still experiencing this on v0.1.23.2 or higher, and it's not just the auto-capitalization setting being enabled and capitalizing the first letter, please provide more info like the specific app and text field where it happens.

Also keep in mind the shifted indicator looks like this

Unshifted
![image](https://github.com/user-attachments/assets/578a65d9-ac25-4144-b255-a58da584bee8)



Shifted
![image](https://github.com/user-attachments/assets/6e79f3d0-e0ee-4488-a716-ab1decea7e4c)
